### PR TITLE
docs: extend examples in find api (#473)

### PR DESCRIPTION
**Pull request by igeligel**
Regarding #473 i updated the documentation of the `.find(selector)` function. Whether the proposed API will be implemented, I guess the `.find(selector)` function stays the same more or less and documentation about getting a component should include all possibilities.

I included examples of every selector possible so people might see it. I also included some whitespace so the different types of selections are separated.

Thanks for the great project 👍

### CODE REVIEW

**Comment by 38elements** [comment]
IMHO,
Since it seems that `name` option and `ref` option are the same,
I think that it is difficult to understand the difference between `name` option  and `ref` option.

File: docs/en/api/wrapper/find.md [hunk 1]
```diff
@@ -17,10 +17,18 @@ import Foo from './Foo.vue'
...
 const div = wrapper.find('div')
 expect(div.is('div')).toBe(true)
+
 const bar = wrapper.find(Bar)
 expect(bar.is(Bar)).toBe(true)
+
+const barByName = wrapper.find({ name: 'bar' })
+expect(barByName.is(Bar)).toBe(true)
+
+const barRef = wrapper.find({ ref: 'bar' })
```
Comment (eddyerburgh) [inline docs/en/api/wrapper/find.md]: Can you  change this to a different value, like  `'foo'`

**Review by eddyerburgh — CHANGES_REQUESTED** [review]

File: docs/en/api/wrapper/find.md (hunk 1 above)
Comment (igeligel) [reply docs/en/api/wrapper/find.md]: I renamed this now to `foo` but I guess it is somehow confusing.

**Review by eddyerburgh — APPROVED** [review]
Thanks 😀

**Pull request merged**